server/src-exec/Migrate.hs

```diff
@@ -1,3 +1,9 @@
+{- |
+Description: Migrations for Hasura metadata.
+-}
+
+{-# LANGUAGE ConstraintKinds #-}
```

Comment:
Add `ConstraintKinds` to `default-extensions` in `graphql-engine.cabal` file. And remove it elsewhere in the server code.